lua/go/inlay.lua

```diff
@@ -112,10 +112,8 @@ local function parseHints(result)
       end
     end
 
-    if only_current_line then
-      if line == tostring(current_line - 1) then
+    if only_current_line and line == current_line - 1 then
```

Comment:
I think the issue is tostring.It should be this:
```
   if only_current_line then
      if line == current_line - 1 then
        add_line()
      end
    else
      add_line()
    end
```

Reply:
no problem